Explain new triggers and conditions - part I (#44422)
home-assistant/home-assistant.github.io: source/_docs/automation/basics.markdown

State changes can be used as the source of triggers and the current state can be used in conditions.

To explore the available _actions_, open the {% my developer_services title="**Settings** > **Developer tools** > **Actions**" %}. _Actions_ allow changing anything. For example, turn on a light, run a script, or enable a scene. Each _action_ has a domain and a name. For example, the _action_ {% my developer_call_service service="light.turn_on" %} is capable of turning on any light in your system. Parameters can be passed to an _action_ to indicate, for example, which device to activate or which color to use.

## Creating automations

Now that you've got a sneak peek of what is possible, it's time to get your feet wet and create your first automation.

### [Creating automations without the new Labs features &raquo;](/docs/automation/editor/)

### Creating automations with the new Labs features

> After enabling the automation preview features in Labs, you can create an automation by following the steps below.

Author: If https://github.com/home-assistant/home-assistant.io/pull/44175 is merged, add the include too:

```suggestion
{% include integrations/labs_entity_triggers_note.md %}

After enabling the automation preview features in Labs, you can create an automation by following the steps below.
```

## Creating automations

Now that you've got a sneak peek of what is possible, it's time to get your feet wet and create your first automation.

### [Creating automations without the new Labs features &raquo;](/docs/automation/editor/)

### Creating automations with the new Labs features

{% include integrations/labs_entity_triggers_note.md %}

After enabling the automation preview features in Labs, you can create an automation in the visual editor of the UI by following the steps below.

#### Adding a trigger

1. Go to **Settings** > **Automations & scenes**.
2. In the lower right corner, select **Create automation** > **Create new automation**.
3. In the **When** section, select **Add trigger**.
4. Now you can do one of the following:

>    - _Select a target and then a trigger._

Author: Steps at this level don't need emphasis:

```suggestion
   - Select a target and then a trigger.
```

Same for the ones below.

     - Under **By target**, select the entity, device or service to which you want to apply the trigger to. You can find the available entities, devices or services listed by:
       - Area of your home, in the **Home** section.
       - **Entities**, **Devices** and **Services**, in the **Unassigned** section.
       - Labels that you have previously created, in the **Labels** section.
     - For that target, you can see the available triggers on the right side. Select **+** on the desired one.
   - _Select a trigger._
     - Select **By type**, and then select the trigger type from the list on the left.

>      - From the listed triggers, select the **+** on the one you choose.

Author: These should be in an ordered list:

```suggestion
     1. Select **By type**, and then select the trigger type from the list on the left.
     2. From the listed triggers, select the **+** on the one you choose.
```

   - _Search for a trigger or target and then select a trigger._
     - Enter the name of one of the following in the search box:
       - A target: an entity, a device or a service.
       - A group of targets: an area or label.
       - A trigger.

>      - From the listed results, select **+** on the desired trigger.

Author: ```suggestion
     1. Enter the name of one of the following in the search box:
        - A target: an entity, a device or a service.
        - A group of targets: an area or label.
        - A trigger.
     2. From the listed results, select **+** on the desired trigger.
```

> 5. In the window on the right, you will have different options depending on the selected trigger. You may find the following ones, for example:

Author: It's better to use "might" to mean possibility to avoid confusion with giving permission:

```suggestion
5. In the window on the right, you will have different options depending on the selected trigger. You might find the following ones, for example:
```

   - Under **Targets**, you can select the first target or another one by selecting **Add target**.
     - Select an entity, device or service to monitor a specific one.
     - Select entities, devices or services in an area, floor or with a certain label to monitor a group of them.

>      - You can add more than one target by selecting **Add target** again.

Author: Use a numbered list here:

```suggestion
     1. Select an entity, device or service to monitor a specific one.
     2. Select entities, devices or services in an area, floor or with a certain label to monitor a group of them.
     3. You can add more than one target by selecting **Add target** again.
```

State changes can be used as the source of triggers and the current state can be used in conditions.

To explore the available _actions_, open the {% my developer_services title="**Settings** > **Developer tools** > **Actions**" %}. _Actions_ allow changing anything. For example, turn on a light, run a script, or enable a scene. Each _action_ has a domain and a name. For example, the _action_ {% my developer_call_service service="light.turn_on" %} is capable of turning on any light in your system. Parameters can be passed to an _action_ to indicate, for example, which device to activate or which color to use.

## Creating automations

Now that you've got a sneak peek of what is possible, it's time to get your feet wet and create your first automation.

> ### [Creating automations without the new Labs features &raquo;](/docs/automation/editor/)

Author: As we saw, this breaks the link in the right nav. I suggest not making headings links and instead describing this in the body. We can even drop this heading altogether:

```suggestion
By default, to create automations, use the [automation editor](/docs/automation/editor/).
```

(or whatever wording works better)

Author: Coincidently, this was a similar problem: https://github.com/home-assistant/home-assistant.io/pull/44560

Reviewer: Done!
